lobsterpy/cohp/test/test_describe.py

```diff
@@ -400,78 +426,129 @@ def test_text(self):
+                "K2 has a 4-fold coordination environment. It has 4 K-Sb (mean ICOHP: -0.36 eV, "
+                "4.969 percent antibonding interaction below EFermi) bonds.",
             ],
         )
         self.assertEqual(
             self.describe_K3Sb_all.text,
             [
                 "The compound K3Sb has 3 symmetry-independent atoms(s) with relevant bonds: K1, K2, Sb4.",
-                "K1 has a 14-fold coordination environment. It has 8 K-K (mean ICOHP: -0.37 eV, 17.544 percent antibonding interaction below EFermi), and 6 K-Sb (mean ICOHP: -0.14 eV, 2.299 percent antibonding interaction below EFermi) bonds.",
-                "K2 has a 14-fold coordination environment. It has 10 K-K (mean ICOHP: -0.22 eV, 17.073 percent antibonding interaction below EFermi), and 4 K-Sb (mean ICOHP: -0.36 eV, 4.969 percent antibonding interaction below EFermi) bonds.",
-                "Sb4 has a 14-fold coordination environment. It has 14 Sb-K (mean ICOHP: -0.27 eV, 3.731 percent antibonding interaction below EFermi) bonds.",
+                "K1 has a 14-fold coordination environment. It has 8 K-K (mean ICOHP: -0.37 eV, 17.544 percent "
+                "antibonding interaction below EFermi), and 6 K-Sb (mean ICOHP: -0.14 eV, 2.299 percent "
+                "antibonding interaction below EFermi) bonds.",
+                "K2 has a 14-fold coordination environment. It has 10 K-K (mean ICOHP: -0.22 eV, 17.073 "
+                "percent antibonding interaction below EFermi), and 4 K-Sb (mean ICOHP: -0.36 eV, 4.969 "
+                "percent antibonding interaction below EFermi) bonds.",
+                "Sb4 has a 14-fold coordination environment. It has 14 Sb-K (mean ICOHP: -0.27 eV, 3.731 "
+                "percent antibonding interaction below EFermi) bonds.",
             ],
         )
         self.assertEqual(
             self.describe_CsH_all.text,
             [
                 "The compound CsH has 2 symmetry-independent atoms(s) with relevant bonds: Cs1, H2.",
-                "Cs1 has a 18-fold coordination environment. It has 18 Cs-Cs (mean ICOHP: -0.49 eV, 18.741 percent antibonding interaction below EFermi) bonds.",
+                "Cs1 has a 18-fold coordination environment. It has 18 Cs-Cs (mean ICOHP: -0.49 eV, 18.741 "
+                "percent antibonding interaction below EFermi) bonds.",
                 "H2 has a 0 coordination environment. It has 0 bonds.",
             ],
         )
+        self.assertEqual(
+            self.describe_BaTiO3_orb.text,
+            [
+                "The compound BaTiO3 has 1 symmetry-independent cation(s) with relevant cation-anion interactions: Ti2.",
+                "Ti2 has an octahedral (CN=6) coordination environment. It has 6 Ti-O (mean ICOHP: -3.54 eV, "
+                "1.092 percent antibonding interaction below EFermi) bonds.",
+                "The orbitals 2s-4s, 2s-3dxy, 2py-3dxy, 2px-3dx2, 2px-3dxy, 2py-3dyz, 2s-3dz2, 2pz-3dz2, "
```

Comment:
I think I would rotate it the other way around as the sentence before talks about 6 Ti-O bonds.

Reply:
Then all the orbital pairs needs to be reversed.

Reply:
Yes, exactly ;)

Reply:
How do we get plots then? Again reversing everything there? Or I just reverse it before printing only?

Reply:
I think this should be only relevant for the printing and not the plotting output. The plotting output should show the correct bonds as well with simililar labels.

Reply:
Have fixed the inconsistency, need to adapt ref test data now

Reply:
Updated the tests and now ready for re-review 😄